managers/forms.py

```diff
@@ -12,27 +9,27 @@
 from users.models import CustomUser
 
 
-class ManagerSelectMultiple(Select2MultipleWidget):
-    def optgroups(self, name: str, value: List, attrs: Optional[Dict] = None) -> List:
-        self.choices.queryset = CustomUser.objects.exclude(user_type=CustomUser.UserType.EMPLOYEE.name).exclude(
-            pk=self.user_pk if hasattr(self, "user_pk") else None
-        )
-        return super().optgroups(name, value, attrs)
-
-
 class ProjectAdminForm(forms.ModelForm):
     def __init__(self, *args: Any, **kwargs: Any) -> None:
         super().__init__(*args, **kwargs)
-        if hasattr(self, "user_pk"):
-            ManagerSelectMultiple.user_pk = self.user_pk
+
+        if "managers" in self.fields:
```

Comment:
This check doesn't make much sense, as this field is in form by its definition.

Reply:
But manager is able to update project, too. However they don't have rights to update managers in project and  then we are using for them`ProjectManagerForm` which inheritance from `ProjectAdminForm` but `manager` field is excluded in `ProjectManagerForm`

Reply:
Then inheritance doesn't make much sense here any more. It should be used if parent class can contain logic from subclasses so you don't have to repeat this logic. Here now those two forms have different logic.